package.json

```diff
@@ -16,6 +16,12 @@
     "eslint": ">= 4.19.1",
     "eslint-config-standard": "^11.0.0"
   },
+  "dependencies": {
```

Comment:
Ideally I'd like these to be devDependencies, however when they are only devDependencies, yarn install over in web does not pick up the dependencies for some reason.

Reply:
Yeah. It's correct that they're `dependencies`, not `devDependencies`. `devDependencies` indicate dependencies only need for developing the style-guide node module. In this case, these `eslint-*` dependencies are needed by the eslint config _provided_ by the style-guide node module, making them full fledged `dependencies`.